control_panel_api/models.py

```diff
@@ -60,6 +62,18 @@ def aws_delete_role(self):
         services.delete_role(self.aws_role_name)
 
 
+class AppUser(TimeStampedModel):
```

Comment:
Regarding the name of this model, I know in the design diagram we called this `AppUser` but, I always thought about these models in terms of `SubjectObject`.
E.g. `UserS3Bucket` or `AppS3Bucket` because it's the user or the app the subject accessing the S3 bucket.
Shouldn't we call this `UserApp` to be consistent with this?